### PR TITLE
Use configurable URL for OeREB web service

### LINKED ISSUE #227: Add configurable web service URL
For usage in a standalone container, an option to configure a separate URL for the OeREB web service needs to be added.

- Add a new optional configuration parameter `service_url`.
- If defined, the specified URL will be used as root for the `GetEGRID` and `GetExtractById` requests.
- If undefined, the application URL is used (as already implemented).

### PR DESCRIPTION
**Pull request by kdeininger**
Closes #227

### CODE REVIEW

File: oereb_client.yml [hunk 1]
```diff
@@ -56,6 +56,9 @@ oereb_client:
       LAYERS: oereb_published_municipalities
       FORMAT: image/png
 
+  # OeREB service URL (optional)
```
Comment (voisardf) [inline oereb_client.yml:59]: OeREB service base URL (optional)?

File: test/py/test_views_index.py [hunk 2]
```diff
@@ -118,6 +118,7 @@ def test_get_config(mock_request):
         index = Index(mock_request)
         assert index.get_config() == {
             'application_url': 'http://example.com/',
```
Comment (voisardf) [inline test/py/test_views_index.py:120]: is this one still used?

**Review by voisardf — DISMISSED** [review]
a suggestion and a question. looks good to me otherwise

File: test/py/test_views_index.py (hunk 2 above)
Comment (kdeininger) [reply test/py/test_views_index.py:120]: Yes, the application URL is still needed for the search service.

File: oereb_client.yml (hunk 1 above)
Comment (kdeininger) [reply oereb_client.yml:59]: Might be better :+1:

**Review by michmuel — APPROVED** [review]

**Pull request merged**